tools/opt/opt.cpp

```diff
@@ -103,11 +103,18 @@ int main(int argc, char** argv) {
           auto spec_ids_vals =
               opt::SetSpecConstantDefaultValuePass::ParseDefaultValuesString(
                   argv[argi]);
+          if (!spec_ids_vals) {
```

Comment:
Interesting! Forgot the case that `ParseDefaultValuesString` can also return `nullptr`. Thanks for fixing it!